projects/1.20/assets/cybernetic-system/cybernetic_system/lang/zh_cn.json

```
@@ -0,0 +1,899 @@
+  "gui.cybernetic_system.gui_craft_medium_container.label_2": "2",
+  "gui.cybernetic_system.gui_craft_medium_container.label_4": "4",
+  "gui.cybernetic_system.gui_craft_medium_container.tooltip_create": "制造",
+  "gui.cybernetic_system.gui_craft_medium_container.tooltip_small_container": "小型保险箱",
+  "gui.cybernetic_system.gui_craft_rein_plate.label_2": "2",
+  "gui.cybernetic_system.gui_craft_rein_plate.label_21": "2",
+  "gui.cybernetic_system.gui_craft_rein_plate.label_22": "2",
+  "gui.cybernetic_system.gui_craft_rein_plate.label_4": "4",
+  "gui.cybernetic_system.gui_craft_rein_plate.tooltip_create": "制造",
+  "gui.cybernetic_system.gui_craft_sm.label_1": "1",
+  "gui.cybernetic_system.gui_craft_sm.label_20": "20",
+  "gui.cybernetic_system.gui_craft_sm.label_3": "3",
+  "gui.cybernetic_system.gui_craft_sm.label_6": "6",
+  "gui.cybernetic_system.gui_craft_sm.tooltip_create": "制造",
+  "gui.cybernetic_system.gui_craft_sm.tooltip_glass_block": "玻璃块",
+  "gui.cybernetic_system.gui_craft_small_conte.label_1": "1",
+  "gui.cybernetic_system.gui_craft_small_conte.label_11": "1",
+  "gui.cybernetic_system.gui_craft_small_conte.label_2": "2",
+  "gui.cybernetic_system.gui_craft_small_conte.label_3": "3",
+  "gui.cybernetic_system.gui_craft_small_conte.label_5": "5",
+  "gui.cybernetic_system.gui_craft_small_conte.tooltip_create": "制造",
+  "gui.cybernetic_system.gui_craft_tm_1.label_1": "1",
+  "gui.cybernetic_system.gui_craft_tm_1.label_11": "1",
+  "gui.cybernetic_system.gui_craft_tm_1.label_2": "2",
+  "gui.cybernetic_system.gui_craft_tm_1.label_6": "6",
+  "gui.cybernetic_system.gui_craft_tm_1.tooltip_create": "制造",
+  "gui.cybernetic_system.gui_craft_transport.tooltip_create": "制造",
+  "gui.cybernetic_system.gui_crafting_station.tooltip_create": "制造",
+  "gui.cybernetic_system.gui_create_mb_1.label_1": "1",
+  "gui.cybernetic_system.gui_create_mb_1.label_15": "15",
+  "gui.cybernetic_system.gui_create_mb_1.label_2": "2",
+  "gui.cybernetic_system.gui_create_mb_1.label_32": "32",
+  "gui.cybernetic_system.gui_create_mb_1.label_4": "4",
+  "gui.cybernetic_system.gui_create_mb_1.label_5": "5",
+  "gui.cybernetic_system.gui_create_mb_1.tooltip_create": "制造",
+  "gui.cybernetic_system.gui_create_mb_2.label_1": "1",
+  "gui.cybernetic_system.gui_create_mb_2.label_15": "15",
+  "gui.cybernetic_system.gui_create_mb_2.label_2": "2",
+  "gui.cybernetic_system.gui_create_mb_2.label_21": "2",
+  "gui.cybernetic_system.gui_create_mb_2.label_32": "32",
+  "gui.cybernetic_system.gui_create_mb_2.label_4": "4",
+  "gui.cybernetic_system.gui_create_mb_2.label_5": "5",
+  "gui.cybernetic_system.gui_create_mb_2.label_6": "6",
+  "gui.cybernetic_system.gui_create_mb_2.tooltip_create": "制造",
+  "gui.cybernetic_system.gui_diolog_dna.label_here_eat_it_and_youll_get_bett": "拿着！吃了它你会变得超极棒！",
+  "gui.cybernetic_system.gui_diolog_dna.label_receive": "[接受]",
+  "gui.cybernetic_system.gui_diolog_golden_apple.label_an_apple_an_apple_now_bring": "苹果！苹果！现在...给我带DNA过来！",
```

```suggestion
  "gui.cybernetic_system.gui_diolog_golden_apple.label_an_apple_an_apple_now_bring": "苹果！苹果！现在……给我带DNA过来！",
```
使用中文省略号（输入法下<kbd>Shift</kbd>+<kbd>6</kbd>）